src/js/button.js

```diff
@@ -1,7 +1,9 @@
 import Component from './component';
-import * as Lib from './lib';
-import * as Events from './events';
+import * as Dom from './utils/dom.js';
+import * as Events from './utils/events.js';
+import * as Fn from './utils/fn.js';
 import document from 'global/document';
+import assign from 'object.assign';
```

Comment:
The shim does actually check for issues and returns the shimmed version if so. So, there are benefits. But yeah, it's possible for the browser to super optimize things like `Object.assign`.